### ISSUE TITLE
Update Norwegian Nynorsk translations

### ISSUE THREAD

**Issue by audunklyve** [squidfunk/mkdocs-material#6187]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "nn",
  "action.edit": "Rediger denne sida",
  "action.skip": "Gå til innhald",
  "action.view": "Vis kjeldekoden for denne sida ⬅️",
  "announce.dismiss": "Ikkje vis igjen",
  "blog.archive": "Arkiver ⬅️",
  "blog.categories": "Kategoriar ⬅️",
  "blog.categories.in": "i ⬅️",
  "blog.continue": "Les vidare ⬅️",
  "blog.draft": "Utkast ⬅️",
  "blog.index": "Tilbake til index ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Relaterte lenkjer ⬅️",
  "clipboard.copy": "Kopier til utklippstavla",
  "clipboard.copied": "Kopiert til utklippstavla",
  "consent.accept": "Akseptert",
  "consent.manage": "Innstillingar",
  "consent.reject": "Avvis",
  "footer": "Footer",
  "footer.next": "Neste",
  "footer.previous": "Førre",
  "header": "Header",
  "meta.comments": "Kommentarar",
  "meta.source": "Kjelde",
  "nav": "Navigasjon",
  "readtime.one": "1 min lesnad ⬅️",
  "readtime.other": "# min lesnad ⬅️",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed for oppdatert innhald ⬅️",
  "search": "Søk",
  "search.placeholder": "Søk",
  "search.share": "Del",
  "search.reset": "Nullstill",
  "search.result.initializer": "Startar søk",
  "search.result.placeholder": "Skriv søkjeord",
  "search.result.none": "Ingen treff",
  "search.result.one": "1 treff",
  "search.result.other": "# treff",
  "search.result.more.one": "1 til på denne sida",
  "search.result.more.other": "# fleire på denne sida",
  "search.result.term.missing": "Manglar",
  "select.language": "Vel språk",
  "select.version": "Vel versjon",
  "source": "Gå til kjelde",
  "source.file.contributors": "Bidragsytarar ⬅️",
  "source.file.date.created": "Oppretta",
  "source.file.date.updated": "Sist oppdatert",
  "tabs": "Faner",
  "toc": "Innhaldsliste",
  "top": "Tilbake til toppen"
}[key] }}{% endmacro %}
```

### Country flag

:flag_no:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in e20cf0bec.

**Comment by squidfunk** [comment]
Released as part of [9.4.6](https://squidfunk.github.io/mkdocs-material/changelog/#9.4.6).
